README.md

```diff
@@ -67,6 +67,30 @@ quest {uri: 'www.google.com', jar: j}, (err, resp, body) ->
   # The request to Google was sent with the cookie that you specified
 ```
 
+## Promises
+Quest also supports [ES6 Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). However, make sure coffeescript is fully up to date. Older versions of coffeescript (e.g. 1.6.x) require parentheses for quest to correctly return Promise objects
+```coffeescript
```

Comment:
Valid point. I wasn't sure if this was a broader coffeescript issue or an edge case that we simply happen to hit. Will remove before merging.